src/generators/dart.ts

```diff
@@ -12,16 +12,29 @@ const supportedArgs = new Set([
   "form-string",
   "no-compressed",
 ]);
+
 function escape(value: string, quote: '"' | "'"): string {
   // Escape Dart's $string interpolation syntax
   // TODO: does Dart have the same escape sequences as JS?
   return jsesc(value, quote).replace(/\$/g, "\\$");
 }
+
 function reprStr(value: string): string {
   const quote = value.includes("'") && !value.includes('"') ? '"' : "'";
   return quote + escape(value, quote) + quote;
 }
-function repr(value: Word, imports: Set<string>): string {
+
+function repr(value: Word | Word[], imports: Set<string>): string {
```

Comment:
```suggestion
function reprWords(value: Word | Word[], imports: Set<string>): string {
```

and rename `reprForWord` back to `repr()` so that all `repr()` functions in the project have the same signature.